### ISSUE TITLE
"This site can't be reached" error

### ISSUE THREAD

**Issue by ellagraceparker** [NLP-Suite/nlp-suite-runner#1]
When I put the link that is outputted in my terminal in Google Chrome, I get this message. I have tried restarting Docker and my computer.

<img width="1292" alt="Screenshot 2024-04-17 at 1 55 21 PM" src="https://github.com/NLP-Suite/nlp-suite-runner/assets/156967468/e99c89a3-2a7c-4570-b39e-419e3452cf47">

**Comment by RafaelPiloto10** [comment]
@ellagraceparker can you please share a screenshot of the output from the Runner? A terminal like window should pop up with some information.

**Comment by ellagraceparker** [comment]
Just ran it again and this is what the terminal said.
<img width="1077" alt="Screenshot 2024-04-17 at 7 15 00 PM" src="https://github.com/NLP-Suite/nlp-suite-runner/assets/156967468/f47b31ae-36c8-4917-87fd-0a230b2d4b72">

**Comment by ellagraceparker** [comment]
After restarting docker which I had found helped last time this is what was shown. 
<img width="972" alt="Screenshot 2024-04-17 at 7 16 04 PM" src="https://github.com/NLP-Suite/nlp-suite-runner/assets/156967468/ba9ba8c1-59b6-4808-b343-9698d6a5c6d7">

**Comment by RafaelPiloto10** [comment]
In the first screenshot, it seems like the NLP Suite failed to start. On restarting, I see the NLP suite has successfully started. Can you confirm if it now works @ellagraceparker ?

**Comment by ellagraceparker** [comment]
No, I run into the same error when I put the link in google that i posted first that the site cannot be reached

**Comment by RafaelPiloto10** [comment]
Could you please open Docker, head over to "Containers", and then click on the one labeled "NLP Suite Runner", and then take a screenshot of the logs tab?

**Comment by ellagraceparker** [comment]
<img width="1277" alt="Screenshot 2024-04-17 at 7 34 06 PM" src="https://github.com/NLP-Suite/nlp-suite-runner/assets/156967468/fa52c0b3-aff5-4271-bb06-b72daf0413d4">

Which one are you referring to?

**Comment by RafaelPiloto10** [comment]
brave_gooda -- click on that and send a screenshot of the logs. I see it has a status of Exited (1), which means it crashed for some reason for you

**Comment by ellagraceparker** [comment]
<img width="877" alt="Screenshot 2024-04-17 at 9 42 51 PM" src="https://github.com/NLP-Suite/nlp-suite-runner/assets/156967468/19ed4611-f64b-4197-87ff-959d85e3ddb7">

**Comment by RafaelPiloto10** [comment]
@ellagraceparker can you download the latest version of the NLP Suite Runner and try again? Thanks
